testgrid/config-upload.sh

```diff
@@ -18,7 +18,11 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-bazel clean --expunge
-bazel build //testgrid:testgrid-config
-gsutil cp bazel-genfiles/testgrid/testgrid-config gs://k8s-testgrid/config
-gsutil cp bazel-genfiles/testgrid/testgrid-config gs://k8s-testgrid-canary/config
+
+for output in gs://k8s-testgrid-canary/config gs://k8s-testgrid/config; do
+  bazel run //testgrid/cmd/config -- \
+    --yaml="$(dirname "${BASH_SOURCE}")"/config.yaml \
+    --output="${output}" \
+    --print-text \
+    --oneshot
```

Comment:
Do we need to specify a service account here, or will it pick up the right one from the environment?

Reply:
Yes it should pick up the `GOOGLE_APPLICATION_CREDENTIALS` var

Reply:
Kind of a pain to trace down but

https://github.com/GoogleCloudPlatform/google-cloud-go/blob/master/storage/storage.go#L94
https://github.com/google/google-api-go-client/blob/master/transport/http/dial.go#L64
https://github.com/google/google-api-go-client/blob/master/internal/creds.go#L41
https://github.com/golang/oauth2/commit/7af32f14d0a25aec7873e0683e8e48dcead159a8